onbuild/yarn/Dockerfile

```diff
@@ -2,17 +2,8 @@ FROM zenika/alpine-node
 
 RUN npm install -g yarn
 
-# hack for the issue: https://github.com/yarnpkg/yarn/issues/918
-# create a fake user to handle yarn cache process
-RUN adduser -D foo
-RUN chown -R foo /home/foo
-RUN chown -R foo /usr/src/app
-
-USER foo
-# end hack https://github.com/yarnpkg/yarn/issues/918
-
-ONBUILD COPY package.json /usr/src/app/
-ONBUILD RUN yarn --pure-lockfile
+ONBUILD COPY package.json yarn.lock /usr/src/app/
+ONBUILD RUN yarn 
```

Comment:
because we now add the yarn.lock in the container so we can use it. ✌️
```
yarn install --pure-lockfile
> Don’t generate a yarn.lock lockfile.
```
I do not remember the usecase of not generating the `yarn.lock` 😝